frontend/components/Course/InstructorQueuePage/Queue.tsx

```diff
@@ -110,11 +121,31 @@ const Queue = (props: QueueProps) => {
                             />
                         )}
                         {queue.active && (
-                            <Label
-                                content={`${
-                                    queue.staffActive || 0
-                                } staff active`}
-                                icon={<Icon name="sync" loading={true} />}
+                            <Popup
+                                content={
+                                    activeStaff && activeStaff.length > 0 ? (
+                                        <List divided verticalAlign="middle">
```

Comment:
I personally like the aesthetic of List without divided - the line seems a bit weird. Were there any issues if this wasn't divided?